shell/platform/embedder/embedder.h

```diff
@@ -216,7 +216,12 @@ typedef enum {
 } FlutterOpenGLTargetType;
 
 typedef struct {
-  /// Target texture of the active texture unit (example GL_TEXTURE_2D).
+  /// Width of the texture.
+  size_t width;
```

Comment:
Are these changes still needed? I thought the explicit width/height was due to the (now-fixed) Skia issue.

Reply:
@stuartmorgan The explicit `width/height` is due to the need for `GL_TEXTURE_RECTANGLE`. The (now-fixed) Skia issue only solves the rendering failure, but the `GL_TEXTURE_RECTANGLE` type of Texture is not scaled in the `Embedder`, so it does not fill the Texture widget correctly. That kind of the wrong image.

<img width="236" height="187" src="https://user-images.githubusercontent.com/17733746/65658856-f8a94780-e05b-11e9-9842-3314dce9a19e.png"/>

So it seems to me to be necessary because when processing the `GL_TEXTURE_RECTANGLE `texture in `Embedder`, we need to know the actual size of the texture to scale to the size of the bounds, and `GL_TEXTURE_2D` is not needed because it can scale itself.

Another way is to change the width/height parameter in the `TextureFrameCallback` to a pointer or reference. The `TextureFrameCallback` is called and the actual size of the texture is passed, so we don't need to add `width/height` to the `FlutterOpenGLTexture`.